pkg/identityserver/store/api_key_store.go

```diff
@@ -125,12 +154,25 @@ func (s *apiKeyStore) UpdateAPIKey(ctx context.Context, entityID *ttnpb.EntityId
 		}
 		return nil, err
 	}
-	if len(key.Rights) == 0 {
+	// Support for older cli versions which do not set field masks for UpdateAPIKey.
```

Comment:
This has nothing to do with the CLI, it's about API compatibility with previous versions of The Things Stack.